app/assets/javascripts/discourse/controllers/reorder-categories.js.es6

```diff
@@ -0,0 +1,106 @@
+import ModalFunctionality from 'discourse/mixins/modal-functionality';
+import { bufferedProperty } from 'discourse/mixins/buffered-content';
+import binarySearch from 'discourse/lib/binary-search';
+import { popupAjaxError } from 'discourse/lib/ajax-error';
+
+// Modal for editing / creating a category
+export default Ember.Controller.extend(ModalFunctionality, Ember.Evented, {
+
+  rawCategories: function() {
```

Comment:
I believe you can use an alias here: `rawCategories: Ember.computed.alias("site.categories")`